src/django/api/migrations/0134_remove_sources_without_contributor.py

```diff
@@ -1,36 +1,40 @@
 from django.db.migrations import Migration, RunPython
 
-from api.models.source import Source
-from api.models.facility.facility_list import FacilityList
-from api.models.facility.facility_list_item import FacilityListItem
-from api.models.facility.facility_match import FacilityMatch
```

Comment:
Updating early migration looks risky. Is it possible to apply this change on follow-up migration instead? Also, can you briefly describe the purpose of these changes?